src/runtime/formal/ops/ops.cc

```diff
@@ -223,13 +230,11 @@ CVM_REGISTER_GLOBAL("cvm.runtime.formal.max_pool2d")
       for (int s = 0; s < filter_w; ++s) {
         int32_t tp = p * stride_h + r - padding[0];
         int32_t tq = q * stride_w + s - padding[1];
-        int32_t x_tmp = minV;
         if (0 <= tp && tp < x_h && 0 <= tq && tq < x_w) {
```

Comment:
Add comments that reserve the original value for x_tmp, which means the comparison
for `x_tmp` and `INT32_MIN`.